### ISSUE TITLE
max() arg is an empty sequence error on listing empty tasks table

### ISSUE THREAD

**Issue by eugenetriguba** [illBeRoy/taskipy#51]
If I have a pyproject.toml file with an empty `[tool.taskipy.tasks]` table, `taskipy.help.HelpFormatter.print` errors out on the call to `max` with "max() arg is an empty sequence". I'd expect that there should instead be an empty table printed or something like "No tasks."

```
$ ls
pyproject.toml
$ cat pyproject.toml
[tool.taskipy.tasks]
$ task -l
max() arg is an empty sequence
$ echo $?
1
```

```
$ echo 'test_task = "echo 'test'"' >> pyproject.toml
$ cat pyproject.toml
[tool.taskipy.tasks]
test_task = "echo test"
$ task -l
test_task echo test
```

Environment:

```
$ pip list
Package    Version
---------- -------
colorama   0.4.5
pip        22.0.4
psutil     5.9.1
setuptools 58.1.0
taskipy    1.10.2
tomli      2.0.1
```

**Comment by illBeRoy** [comment]
Hey! Thanks for the bug report!
I fixed the bug and opened a PR here: #52 
Please review when you have some free time and I'll merge it once approved :)

**Comment by illBeRoy** [comment]
Fix released as [taskipy 1.10.13](https://pypi.org/project/taskipy/1.10.3/)
